app/controllers/api/v1/api_controller.rb

```diff
@@ -3,7 +3,7 @@ module V1
     class ApiController < ApplicationController
       include Wor::Paginate
       include DeviseTokenAuth::Concerns::SetUserByToken
-      protect_from_forgery with: :null_session
+      protect_from_forgery with: :exception
```

Comment:
Did you test you end point after did this? Are you sure you have understand the difference with ApplicationController and ApiController?